ee/tabby-db/src/threads.rs

```diff
@@ -50,6 +50,7 @@ pub struct ThreadMessageAttachmentWebDoc {
 pub struct ThreadMessageAttachmentIssueDoc {
     pub title: String,
     pub link: String,
+    pub author_email: Option<String>,
```

Comment:
```suggestion
    pub author_user_id: Option<String>,
```

Store author user id directly.